cmd/rpc/client.go

```diff
@@ -48,22 +47,25 @@ func (client *Client) Call(serviceMethod string, args, reply interface{}) error
 		return fmt.Errorf("rpc reply must be a pointer type, but found %v", replyKind)
 	}
 
-	data, err := gobEncode(args)
-	if err != nil {
+	b1 := bufPool.Get()
```

Comment:
Let's use meaningful names like `argsBuf` and `reqBuf` instead of `b1` and `b2`.